docs/distributed/DECOMMISSION.md

```diff
@@ -73,7 +73,7 @@ Stop an on-going decommission in progress, mainly used in situations when the lo
 └─────┴─────────────────────────────────┴──────────────────────────────────┴────────────────────┘
 ```
 
-If for some reason decommission fails in between, the `status` will indicate decommission as failed instead.
+If for some reason decommission fails in process, the `status` will indicate decommission as failed instead.
```

Comment:
If the decommission process fails for any reason, the `status` indicates `failed`.